index.html

```diff
@@ -9,245 +9,247 @@
+                        <h2>Profesional Art Printing Data</h2>
+                        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
 
-                    <ul class="card-Skills-links ">
-                        <li class="b5 ">HTML</li>
-                        <li class="b6 ">Bootstrap</li>
-                        <li class="b7 ">Ruby</li>
-                    </ul>
+                        <ul class="card-Skills-links">
+                            <li class="b5 ">HTML</li>
+                            <li class="b6 ">Bootstrap</li>
+                            <li class="b7 ">Ruby</li>
+                        </ul>
+                    </article>
+                    <button class="card-projectbtn" id="showDetailsBtn">See project</button>
+                </div>
+
+                <div class="fouth-card ">
+                    <article>
+                        <h2>Profesional Art Printing Data</h2>
+                        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+
+                        <ul class="card-Skills-links">
+                            <li class="b5 ">HTML</li>
+                            <li class="b6 ">Bootstrap</li>
+                            <li class="b7 ">Ruby</li>
+                        </ul>
+                    </article>
+                    <button class="card-projectbtn " id="showDetailsBtn">See project</button>
+                </div>
+
+                <div class="fifth-card ">
+                    <article>
+                        <h2>Profesional Art Printing Data</h2>
+                        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+                        <ul class="card-Skills-links ">
+                            <li class="b5 ">HTML</li>
+                            <li class="b6 ">Bootstrap</li>
+                            <li class="b7 ">Ruby</li>
+                        </ul>
+                    </article>
+                    <button class="card-projectbtn " id="showDetailsBtn">See project</button>
+                </div>
+
+                <div class="six-card ">
+                    <article>
+                        <h2>Profesional Art Printing Data</h2>
+                        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+
+                        <ul class="card-Skills-links ">
+                            <li class="b5 ">HTML</li>
+                            <li class="b6 ">Bootstrap</li>
+                            <li class="b7 ">Ruby</li>
+                        </ul>
+                    </article>
+                    <button class="card-projectbtn " id="showDetailsBtn">See project</button>
+                </div>
+            </div>
```

Comment:
You did really well with your desktop popup modal, please kindly implement the following changes 😄 :

-  kindly note that the projects section should be created dynamically with the information from the projects array (currently, all the HTML of the section is created when the page loads). This requires changes in existing code - already created HTML code for the projects list should be removed, and instead, it should be added via JavaScript.

-  The implementation of the popup modal is for both desktop and mobile screens, although you implemented the desktop version in a perfect replica of the Figma design given, please do same for the mobile view using the rightly stipulated design found in the Figma 😄